tests/PartnerClientTest.php

```diff
@@ -67,4 +68,27 @@ public function testChangeSeatsHappyPath()
 
         self::assertNotEmpty($resp, 'expected a response');
     }
+
+    public function testUpdateSSOHappyPath()
+    {
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $partnerClient = new PartnerClient($environment);
+
+        $req = new UpdateSSORequest();
+        $req->setDomain("google.com");
+        // disable SSO
+        $req->setEnableSso(false);
+
+        try {
+            $resp = $partnerClient->UpdateSSO($req);
+        } catch (Vendasta\Vax\SDKException $e) {
+            self::assertEquals(404, $e->getCode(), 'expected a 404');
+            return;
+        }
+
+        self::assertNotEmpty($resp, 'expected a response');
```

Comment:
Shouldn't we assert the response is empty? Or how does that work with google.protobuf.Empty

Reply:
Yes, not sure why my test still passed 👀